redbot/core/dev_commands.py

```diff
@@ -115,7 +115,26 @@ def sanitize_output(ctx: commands.Context, input_: str) -> str:
 
         result = self.sanitize_output(ctx, str(result))
 
-        await ctx.send(box(result, lang="py"))
+        def check(m):
+            return m.content.lower() == "more"
```

Comment:
Just be aware that this will pass for any message "more" sent by any member in any channel which the bot sees. Remember to check that the channel and author match the context.